charts/scalyr-agent/templates/configmap.yaml

```diff
@@ -17,8 +17,25 @@ data:
         {{- if .Values.scalyr.k8s.enableLogs }}
         {
           "module": "scalyr_agent.builtin_monitors.kubernetes_monitor",
+          "stop_agent_on_failure": true,
           "report_container_metrics": {{ .Values.scalyr.k8s.enableMetrics }},
           "report_k8s_metrics": {{ .Values.scalyr.k8s.enableMetrics }}
+        },
+        {{- end }}
+        {{- if .Values.scalyr.k8s.enableExplorer }}
+        {
+          module:  "scalyr_agent.builtin_monitors.kubernetes_openmetrics_monitor",
+          include_node_name: true,
+          include_cluster_name: true,
+          logger_include_node_name: false,
+          // Switch to true once preview phase is over
```

Comment:
Since we've done a stable release, is it time to do that?

Reply:
Functionality is still in preview, but we can decide how to handle it in a future release - for production deployments setting it to true is the correct thing to do, but for minikube setups when people are testing it, they would need to set this to false.

We should definitively set it to true when it goes out of the preview phase and add docs on how to disable it for local testing, minikube and similar.

Reply:
Ah, got it. I just connected a new stable release with the end of the preview.

Reply:
Yep, but I'm 100% on board that the default value should be true once it goes out of the preview :)